src/main/java/uk/gov/hmcts/reform/iacasepaymentsapi/domain/entities/JourneyType.java

```diff
@@ -0,0 +1,35 @@
+package uk.gov.hmcts.reform.iacasepaymentsapi.domain.entities;
+
+import static java.util.Arrays.stream;
+
+import com.fasterxml.jackson.annotation.JsonValue;
+import java.util.Optional;
+
+public enum JourneyType {
```

Comment:
@tarun-palisetty if you could look to include a Test class for this (same as the one in ia-case-notification-api, just to help us with code coverage).